combinetf2/io_tools.py

```diff
@@ -0,0 +1,118 @@
+import h5py
+import numpy as np
+
+from wums import ioutils  # isort: skip
+
+
+def get_fitresult(fitresult_filename, result=None, meta=False):
+    if isinstance(fitresult_filename, str):
+        h5file = h5py.File(fitresult_filename, mode="r")
+    else:
+        h5file = fitresult_filename
+    key = "results"
+    if result is not None:
+        key = f"{key}_{result}"
+    elif key not in h5file.keys():  # fallback in case only asimov was fit
+        key = f"{key}_asimov"
+    h5results = ioutils.pickle_load_h5py(h5file[key])
+    if meta:
+        meta = ioutils.pickle_load_h5py(h5file["meta"])
+        return h5results, meta
+    return h5results
+
+
+def get_poi_names(fitresult):
+    if "impacts" in fitresult.keys():
```

Comment:
this seems like a very indirect way to get the poi names.  Should relevant metadata simply be stored directly in the fitresults?